src/Universe/UniverseGenerator.php

```diff
@@ -0,0 +1,601 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EtoA\Universe;
+
+use DirectoryIterator;
+use EtoA\Core\Configuration\ConfigurationService;
+use EtoA\Support\DatabaseManagerRepository;
+use EtoA\User\UserRepository;
+use Mutex;
+use UserToXml;
+
+/**
+ * @todo Create and use repositories for planets, stars etc.
+ */
+class UniverseGenerator
```

Comment:
Some general thoughts about this class:
* great use of constants and making this readable by moving different parts into methods with helpful names
* it does a lot more than just generating the unvierse e.g. resetting rounds/tables which are note related to the universe. Would probably split this
* It's a bit unfortunate that the for certain things arrays are used instead of objects e.g. coordinates
* this is probably still from the old code but some variables would benefit form being longer than 1-2 chars to make sure you don't have to scroll up to remember what they mean
* there is a bit of a mix of snake vs camel case probably because of ported code

Reply:
Separated reset code b59ebf5

Reply:
Further changes in 9f7c2d7 and 4f26383